src/Magento/FunctionalTestingFramework/Util/Path/FileUrlFormatterInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\FunctionalTestingFramework\Util\Path;
+
+use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
+
+interface FileUrlFormatterInterface
```

Comment:
Change this to `FormatterInterface`

Reply:
Changed

Reply:
What benefit do we get from having this interface with only one function?

Reply:
To me this interface is just to group two type of formatter class. They have similarity but unrelated at the same time. What's the rule for interface?

Reply:
I'm no OOP expert. I just don't like the existence of useless things to exist.

I see the value of an interface being when you have multiple implementations and multiple functions within.

 I just don't like it but probably someone else is best to make the final call on this.